outlines/react.md

```diff
@@ -0,0 +1,34 @@
+# React
+
+1. Introduction to React
+  1. Background: developed by Facebook, used widely in production, vibrant ecosystem
+  2. Installing and using React in Meteor 1.3
+  3. A simple example
+
+2. JSX in depth
+  1. What JSX really is (helping people understand it)
+  2. JSX workflow (transpiling - we've got it covered for you!)
+  3. JavaScript expressions inside JSX
+  4. Conditional and list rendering with JSX
+  5. Gotchas
+
+3. Components in depth
+  1. State
```

Comment:
This is super minor, but I would maybe put state after props, since it's something you're meant to use judiciously these days?